hmpps-sqs-spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -32,6 +32,7 @@ dependencies {
   implementation("io.awspring.cloud:spring-cloud-aws-sqs")
   implementation("com.google.code.gson:gson:2.11.0")
   implementation("com.microsoft.azure:applicationinsights-core:3.5.3")
+  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:2.5.0")
```

Comment:
Please review https://github.com/ministryofjustice/dps-gradle-spring-boot/pull/332/files and check that this is the correct version to import

Reply:
Thanks @markreesmoj - decided to get rid of the version altogether, as it's easy enough to create the Span manually without the annotation.

Reply:
This is great, an annoying dependency we no longer have to maintain in our project